site/docs/concepts/vault_plugins.md

```diff
@@ -23,7 +29,7 @@ vault policy write "allow-all-dataset-creds" - <<EOF
       }
       EOF
 ```
-3. Have the `CatalogDatasetInfo` structure from the [data catalog response](../../reference/connectors#data_catalog_responseproto) contain the Vault secret path which should be used to retrieve the credentials for a given asset. When Vault plugin is used to retrieve the credentials the parameters to the plugin should follow the plugin usage instructions. This path will later be passed on to the [modules](./modules.md).
+4. Have the `CatalogDatasetInfo` structure from the [data catalog response](../../reference/connectors#data_catalog_responseproto) contain the Vault secret path which should be used to retrieve the credentials for a given asset. When Vault plugin is used to retrieve the credentials the parameters to the plugin should follow the plugin usage instructions. This path will later be passed on to the [modules](./modules.md).
```

Comment:
"When Vault plugin" ->"When the Vault plugin"

Reply:
Fixed. Thanks